README.md

```diff
@@ -1,98 +1,144 @@
 # vcenter-connector
 
-vcenter-connector is an OpenFaaS event connector built to consume events from vCenter from VMware
+vcenter-connector is an OpenFaaS event connector built to consume events from vCenter from VMware vSphere.
```

Comment:
`vCenter from VMware vSphere` this sounds confusing. does `from vCenter` work too?